examples/realtime/lslclient_rt_plot.py

```diff
@@ -0,0 +1,86 @@
+"""
+==============================================================
+Plot real-time epoch data with LSL client
+==============================================================
+
+This example demonstrates how to use the LSL client to plot real-time
+collection of event data from an LSL stream.
+For the purposes of demo, a mock LSL stream is constructed. You can
+replace this with the stream of your choice by changing the host id to
+the desired stream.
+
+"""
+# Author: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
+from multiprocessing import Process
+import time
+from random import random as rand
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mne.realtime import LSLClient
+
+print(__doc__)
+
+# this is the host id that identifies your stream on LSL
+host = 'mne_stream'
+# this is the max wait time in seconds until client connection
+wait_max = 5
+
+
+# this is a helper function that will simulate an LSL stream
+def _start_mock_lsl_stream(host):
```

Comment:
do you think we could hide this complexity from the user by having a function in the realtime module which does it for them?

Reply:
are you suggesting creating a `MockLSLClient`? I could follow up with one and update the example. Just don't want feature creep with this PR

Reply:
umm ... just a function would be good enough probably. Something to hide the complexity at the user level. And you could reuse that in your test.

Reply:
If you don't want to do it in this PR, I suggest removing the example and adding it back cleanly in a follow up PR

Reply:
you did ask for an example ;) I actually don't think that it is complex, it's based off the example that LSL uses in their package, all I did was just make the signal a little more interesting. if you think that it would distract from the example, then I will just remove it. my next pr is to do the refactor and include a mocklslclient (if we think that we should have one). 

I don't think just adding a function to the codebase is useful alone just for the example. I would rather make a proper mock client where it is tested just like the real client